package.json

```diff
@@ -84,13 +84,13 @@
     "istanbul-instrumenter-loader": "3.0.1",
     "jasmine": "2.99.0",
     "jasmine-spec-reporter": "4.2.1",
-    "karma": "3.1.3",
+    "karma": "3.1.1",
```

Comment:
Had to downgrade karma to `3.1.1` in order to fix the "watch" bug @Blackbaud-PaulCrowder reported last week.
See: https://github.com/webpack-contrib/karma-webpack/issues/100